service/src/main/resources/api/service_openapi.yaml

```diff
@@ -849,6 +849,47 @@ paths:
           $ref: '#/components/responses/PermissionDenied'
         '500':
           $ref: '#/components/responses/ServerError'
+  /api/workspaces/v1/{workspaceId}/resources/controlled/gcp/buckets/{resourceId}/clone:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+      - $ref: '#/components/parameters/ResourceId'
+    post:
+      summary: Clone a GCS Bucket
+      operationId: cloneGcsBucket
+      tags: [ControlledGcpResource]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CloneControlledGcpGcsBucketRequest'
+      responses:
+        '200':
+          $ref: '#/components/responses/ClonedControlledGcpGcsBucketResponse'
+        '403':
```

Comment:
nit: I think this can also return 202 with the same response body as 200

Reply:
good catch